packages/api/src/controllers/auth/user/UserController.ts

```diff
@@ -73,6 +73,7 @@ export class AccountController {
         statusViewMode: data.statusViewMode as StatusViewMode,
         tableActionsAlignment: data.tableActionsAlignment as TableActionsAlignment,
         soundSettingsId,
+        locale: data.locale || null,
```

Comment:
Note to self: add `locale` cookie and `theme` cookie so the client can read it even if the user isn't authenticated. The browser will keep it saved.